difflint/lib.py

```diff
@@ -2,24 +2,44 @@
 from datetime import datetime
 from difflib import unified_diff
 from io import StringIO
-from os import unlink
+from os import getcwd, unlink
+import os.path
```

Comment:
`path` is a module and the others are functions. Personally I prefer just importing the modules (`import os, os.path`) and using the fully qualified names in the code, since I think it makes the code easier to read, but I'll leave it up to the maintainer